.env.sample

```diff
@@ -0,0 +1,20 @@
+#Tinny ENV Vars
+MAX_ATTEMTPS=1
+NETWORK=localchain
+DEBUG=true
+WAIT_FOR_KEY_INTERVAL=3000
+LIT_OFFICAL_RPC=https://chain-rpc.litprotocol.com/http
+TIME_TO_RELEASE_KEY=10000
+RUN_IN_BAND=true
+RUN_IN_BAND_INTERVAL=5000
+PRIVATE_KEYS="0x59c6995e998f97a5a0044966f0945389dc9e86dae88c7a8412f4603b6b78690d,0x5de4111afa1a4b94908f83103eb1f1706367c2e68ca870fc3fb9a804cdab365a,0x7c852118294e51e653712a81e05800f419141751be58f605c371e15141b007a6,0x47e179ec197488593b187f80a00eb0da91f1b9d0b13f8733639f19c30a34926a,0x8b3a350cf5c34c9194ca85829a2df0ec3153be0318b5e2d3348e872092edffba,0x92db14e403b83dfe3df233f83dfa3a0d7096f21ca9b0d6d6b8d88b2b4ec1564e,0x4bbbf85ce3377467afe5d46f804f221813b2bb87f24d81f60f1fcdbf7cbf4356,0xdbda1821b80551c9d65939329250298aa3472ba22feea921c0cf5d620ea67b97,0x2a871d0798f97d79848a013d4936a73bf4cc922c825d33c1cf7073dff6d409c6"
```

Comment:
this is unrelated to this PR but just a note - i loaded 10 wallets up and put them into a github secret `CHRONICLE_TEST_PRIVATE_KEYS` that can be accessed from a github action.  this way, we can have secret wallets that are funded, and nobody can drain those accounts and make our tests fail.  

we prob need to modify the workflow file to just load the `secrets.CHRONICLE_TEST_PRIVATE_KEYS` key into the `PRIVATE_KEY` env var and then it should "just work".  

anyway just noting this.

Reply:
Making a task for this.